api/src/firstrun.py

```diff
@@ -104,96 +104,98 @@ def create_admin(admins):
+        firstname="first2", lastname="last2", price_level="normal"
     )
-    get_or_create(
-        Member,
+    create_member(
         email="first3.last3@gmail.com",
-        defaults=dict(
-            firstname="first3", lastname="last3", price_level="normal", pending_activation=False, member_number=2002
-        ),
+        firstname="first3", lastname="last3", price_level="low_income_discount"
     )
 
 
 def create_shop_products():
-    banner(BLUE, "Creating Fake Shop Categories and Products")
+    banner(BLUE, "Creating Fake Shop Categories")
 
     display_order_category = db_session.query(func.max(ProductCategory.display_order)).scalar() or 0
     member_category = get_or_create(
-        ProductCategory, name="Medlemskap", defaults=dict(display_order=display_order_category + 1)
+        ProductCategory, name="Membership", defaults=dict(display_order=display_order_category + 1)
     )
     consumption_category = get_or_create(
-        ProductCategory, name="Förbrukning", defaults=dict(display_order=display_order_category + 2)
+        ProductCategory, name="Consumption", defaults=dict(display_order=display_order_category + 2)
     )
     tools_category = get_or_create(
-        ProductCategory, name="Verktyg", defaults=dict(display_order=display_order_category + 3)
+        ProductCategory, name="Tools", defaults=dict(display_order=display_order_category + 3)
     )
     others_category = get_or_create(
-        ProductCategory, name="Övrigt", defaults=dict(display_order=display_order_category + 4)
+        ProductCategory, name="Other", defaults=dict(display_order=display_order_category + 4)
     )
 
+    banner(BLUE, "Creating Fake Shop Products")
+
     display_order_product = db_session.query(func.max(Product.display_order)).scalar() or 0
     prod1 = get_or_create(
         Product,
         name="Base membership",
         defaults=dict(
             price=200,
-            id=62,
             unit="år",
             display_order=display_order_product + 1,
             category_id=member_category.id,
             product_metadata={
-                "allowed_price_levels": ["low_income_discount"],
-                "special_product_id": ["single_membership_year"],
+                "allowed_price_levels": "low_income_discount",
+                "special_product_id": "single_membership_year",
             },
         ),
     )
     get_or_create(ProductAction, product_id=prod1.id, value=365, action_type="add_membership_days")
+
     prod2 = get_or_create(
         Product,
         name="Makerspace access",
         defaults=dict(
             price=575,
-            id=63,
             unit="mån",
             display_order=display_order_product + 2,
             category_id=member_category.id,
             product_metadata={
-                "allowed_price_levels": ["low_income_discount"],
-                "special_product_id": ["single_membership_year"],
+                "allowed_price_levels": "low_income_discount",
```

Comment:
I'm pretty sure this should still be an array.

Reply:
Yes!